REUSE.toml

```diff
@@ -4,32 +4,23 @@ SPDX-PackageSupplier = "SAP OpenUI5 <openui5@sap.com>"
 SPDX-PackageDownloadLocation = "https://github.com/UI5/cli"
 SPDX-PackageComment = "The code in this project may include calls to APIs (“API Calls”) of\n SAP or third-party products or services developed outside of this project\n (“External Products”).\n “APIs” means application programming interfaces, as well as their respective\n specifications and implementing code that allows software to communicate with\n other software.\n API Calls to External Products are not licensed under the open source license\n that governs this project. The use of such API Calls and related External\n Products are subject to applicable additional agreements with the relevant\n provider of the External Products. In no event shall the open source license\n that governs this project grant any rights in or to any External Products,or\n alter, expand or supersede any terms of the applicable additional agreements.\n If you have a valid license agreement with SAP for the use of a particular SAP\n External Product, then you may make use of any API Calls included in this\n project’s code for that SAP External Product, subject to the terms of such\n license agreement. If you do not have a valid license agreement for the use of\n a particular SAP External Product, then you may only make use of any API Calls\n in this project for that SAP External Product for your internal, non-productive\n and non-commercial test and evaluation of such API Calls. Nothing herein grants\n you any rights to use or access any SAP External Product, or provide any third\n parties the right to use of access any SAP External Product, through API Calls."
 
+# Default Apache-2.0 license for everything
 [[annotations]]
 path = "**"
 precedence = "aggregate"
 SPDX-FileCopyrightText = "2025 SAP SE or an SAP affiliate company and UI5 CLI contributors"
 SPDX-License-Identifier = "Apache-2.0"
 
-[[annotations]]
-path = "packages/server/lib/middleware/serveIndex/**"
-precedence = "aggregate"
-SPDX-FileCopyrightText = ["2010 Sencha Inc.", "2011 LearnBoost", "2011 TJ Holowaychuk", "2014-2015 Douglas Christopher Wilson"]
-SPDX-License-Identifier = "MIT"
-
-[[annotations]]
-path = "packages/server/lib/middleware/testRunner/**"
-precedence = "aggregate"
-SPDX-FileCopyrightText = "2025 SAP SE or an SAP affiliate company and OpenUI5 contributors"
-SPDX-License-Identifier = "Apache-2.0"
-
+# Special exceptions - different copyright holder
```

Comment:
The package-specific REUSE.toml files are aggregated when linting the whole repository. So I would assume that the root REUSE.toml file won't include duplicate exceptions for files within a package. The annotations for ui5-server have been removed, but for builder and documentation they are kept. Why is this necessary?

Reply:
Done. Thanks